std/conv.d

```diff
@@ -4015,7 +4472,14 @@ if (isInputRange!Source && isSomeChar!(ElementType!Source))
 
     s.popFront();
 
-    return result;
+    static if (doCount)
+    {
+        return tuple(cast (dchar) result, ++count);
```

Comment:
Forgot the named fields

Reply:
@edi33416 I modified this, thank you for catching the mistake. Can this be marked as resolved?